Server/Content/Sources/App/Controllers/TextController.swift

```diff
@@ -0,0 +1,29 @@
+import Fluent
+import Vapor
+
+struct TextController {
+    func all(req: Request) throws -> EventLoopFuture<[Text]> {
+        return Text.query(on: req.db).all()
+    }
+    
+    func getById(req: Request) throws -> EventLoopFuture<Text> {
+        return Text.find(req.parameters.get("textID"), on: req.db)
+            .unwrap(or: Abort(.notFound))
+    }
+
+    func create(req: Request) throws -> EventLoopFuture<Text> {
+        let text = try req.content.decode(Text.self)
+        let words = text.text.split(separator: " ")
+        let backwardWords = words.map{String($0.reversed())}
+        text.backwardText = backwardWords.joined(separator:" ")
```

Comment:
This algorithm is quite simple to be honest 
Imagine if you have text like:
```
This was the beginning of the story.
```
Output will be:
```
sihT saw eht gninnigeb fo eht .yrots
```
Which is not exactly the desired behavior, because we want for full-stop sign at the end of the sentence stay there, as well as other punctuation symbols.